### ISSUE TITLE
0.1.928 give me a errors for ClassLevelParams

### ISSUE THREAD

**Issue by SimonCropp** [thomhurst/TUnit#743]
```
17>ClassLevelParams__WithMethodLevel__345bf30dcfcc4c2eb621dff2eafb15b3.Generated.cs(20,99): Error CS1729 : 'ClassLevelParams' does not contain a constructor that takes 2 arguments
17>ClassLevelParams__IgnoreParameters__99b08937cd1d42e4bf7ba072e8b8ede7.Generated.cs(20,99): Error CS1729 : 'ClassLevelParams' does not contain a constructor that takes 2 arguments
17>ClassLevelParams__WithMethodLevel__f9c6e1ff0a1f4d7a8c5328a4d8b35da0.Generated.cs(44,104): Error CS1501 : No overload for method 'WithMethodLevel' takes 2 arguments
17>ClassLevelParams__WithMethodLevel__dee01b72776744e3996ba1dfaf38ef00.Generated.cs(20,99): Error CS1729 : 'ClassLevelParams' does not contain a constructor that takes 2 arguments
17>ClassLevelParams__WithMethodLevel__dee01b72776744e3996ba1dfaf38ef00.Generated.cs(44,104): Error CS1501 : No overload for method 'WithMethodLevel' takes 2 arguments
17>ClassLevelParams__IgnoreParameters__1e2d8f1fa04840b690ecab0eb082be10.Generated.cs(44,104): Error CS1501 : No overload for method 'IgnoreParameters' takes 2 arguments
17>ClassLevelParams__IgnoreParameters__f653aaac34384fc2a926e7b2d47c1f7d.Generated.cs(20,99): Error CS1729 : 'ClassLevelParams' does not contain a constructor that takes 2 arguments
17>ClassLevelParams__IgnoreParameters__f653aaac34384fc2a926e7b2d47c1f7d.Generated.cs(44,104): Error CS1501 : No overload for method 'IgnoreParameters' takes 2 arguments
17>------- Finished building project: Verify.TUnit.Tests. Succeeded: False. Errors: 16. Warnings: 0
```

```
#pragma warning disable CS9113 // Parameter is unread.
[Arguments("1")]
[Arguments("2")]
public class ClassLevelParams(string arg1)
{
    [Test]
    public Task Simple() => Verify("Value");

    [Test]
    [Arguments("3")]
    [Arguments("4")]
    public Task WithMethodLevel(string arg2) => Verify("Value");

    [Test]
    [Arguments("3")]
    [Arguments("4")]
    public Task IgnoreParameters(string arg2) =>
        Verify("Value")
            .IgnoreParametersForVerified();
}
```

**Comment by thomhurst** [comment]
Thanks @SimonCropp  - Sorry about that.

Should be resolved in 0.1.931.

**Comment by SimonCropp** [comment]
no worries. and thanks
